src/Spice86.Core/Emulator/CPU/CfgCpu/ParsedInstruction/CfgInstruction.cs

```diff
@@ -39,14 +39,14 @@ protected CfgInstruction(SegmentedAddress address, InstructionField<ushort> opco
         }
 
         OpcodeField = opcodeField;
-        FieldsInOrder.AddRange(PrefixFields);
-        FieldsInOrder.Add(OpcodeField);
+        AddFields(PrefixFields);
+        AddField(OpcodeField);
     }
 
     /// <summary>
     /// To call after constructor to calculate instruction length
     /// </summary>
-    public void PostInit() {
+    private void UpdateLength() {
```

Comment:
Can't the constructor call it ?

Reply:
This is actually what the constructor is doing via the AddField method. Problem is instruction subclasses may also add fields and this could be an issue.